CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Braintree iOS SDK Release Notes
 
+## unreleased
+* BraintreePayPalNativeCheckout
+  * Rename `BTPayPalNativeError` to `BTPayPalNativeCheckoutError`
+  * Add `BTPayPalNativeCheckoutError.deallocated` = 11
+* BraintreeAmericanExpress
+  * Add `BTAmericanExpressError.deallocated` = 2
+* BraintreeCore
+  * Add `BTAPIClientError.deallocated` = 2
+  * Add `BTHTTPError.deallocated` = 12
+* BraintreePayPal
+  * Add `BTPayPalError.deallocated` = 9
+* BraintreeThreeDSecure
+  * Add `BTThreeDSecureError.deallocated` = 8
+* BraintreeSEPADirectDebit
+  * Add `BTSEPADirectDebitError.deallocated` = 8
```

Comment:
Super nitpick / idea - the error codes throughout the CHANGELOG entries for V6 betas get a little unruly/hard to read IMO.

Maybe we could amend the previous entries to remove all error code specific ones, and then instead summarize them in a single table that we include in the V6 GA release notes?

Reply:
I like that idea. Agree right now the changes to the errors is quite hard to reason about. I created internal ticket DTBTSDK-2788 for this work.